README.md

```diff
@@ -190,6 +190,16 @@ end
 
 Scripttags are created in the same way as the Webhooks, with a background job which will create the required scripttags.
 
+Customising Queue Names
+-----------------------------
+By default ActiveJob is using `:default` as queue names. You can override this value in the initializer :
+```ruby
+ShopifyApp.configure do |config|
+  config.scripttags_manager_queue_name = :'my-custom-name-worker'
+  config.webhooks_manager_queue_name = :'another-custom-name-worker'
+```
+
+
```

Comment:
I'd prefer to not call this out in the readme as its pretty specific. The readme is already pretty big - maybe we need a wiki or something else on this repo now.

I think this feature is easy enough for someone to discover if they need it by reading the source